frontend/src/scenes/trends/persons-modal/PersonsModalV2.tsx

```diff
@@ -317,3 +310,9 @@ export const openPersonsModal = (props: OpenPersonsModalProps): void => {
     document.body.appendChild(div)
     ReactDOM.render(<PersonsModalV2 {...props} onAfterClose={destroy} />, div)
 }
+
+export const shouldUsePersonsModalV2 = (): boolean => {
+    const featureFlags = featureFlagLogic.findMounted()?.values?.featureFlags
+
+    return !featureFlags || !featureFlags[FEATURE_FLAGS.PERSONS_MODAL_V2]
```

Comment:
Confusing

When `featureFlags[FEATURE_FLAGS.PERSONS_MODAL_V2]` is `true`, this returns `false` ?

Reply:
I think this was fixed whilst you were reviewing